cider/tests/utils/QueryDataGenerator.h

```diff
@@ -33,6 +33,22 @@
 
 enum GeneratePattern { Sequence, Random, Special_Date_format_String };
 
+// A date within [1000-01-01..9999-12-31].
```

Comment:
I am afraid this range is wrong.

Reply:
this range is defined from substrait. https://github.com/substraitio/substrait/blob/main/site/docs/types/type_classes.md#simple-types
let me replace with our actual generated test data range.